services/registry/.env-example

```diff
@@ -1,7 +1,7 @@
 NODE_ENV=dev
 DEV_LATENCY_ERROR_MS=10000
 REDIS_URL=redis://redis:6379
-PORT=3000
+PORT=3002
```

Comment:
Noted via another channel: the expected port layout is 3000 for registry, 3001 for web, 3002 for storage.